File: practices/financial_calculator.py
#LS, Financial calculator


monthly_income = float (input("What is your monthly income?: "))

rent = float (input("What is your monthly rent/mortgage?: "))

utilities = float(input("What is your monthly utilites cost?: "))

groceries = float(input("What is your monthly spend on groceries?: "))

transportation = float(input("What is your monthly spend on transportation?: "))

rent_precent = (rent / monthly_income) * 100

utilities_precent = (utilities / monthly_income) * 100

groceries_precent = (groceries / monthly_income) * 100

transportation_precent = (transportation / monthly_income) * 100

savings = monthly_income * 0.10 

spending_money = monthly_income - (rent + utilities + groceries + transportation)

print("Your monthly income is $", monthly_income, "which is", rent_precent  % "of your income")
print("Your utilites are", utilities, "which is", utilities_precent % "of your income")
print("Your groceries are", groceries, "which is", groceries_precent % "of your income")
print("Your transportation are", transportation, "which is", transportation_precent % "of your income")
print("You have $", spending_money, "of spending money each month!")

File: C/practices/financial_calculator.c
// LS finacial calculator
#include <stdio.h>
#include <math.h>
int main(){
    int income = printf("What is your monthly income: \n");
    scanf("%d", income);
    int utilities = printf("What is your monthly utilities spend: \n");
    scanf("%d", utilities);
    int groceries = printf("What is your monthly spend on groceries: \n");
    scanf("%d", groceries);
    int transportation = printf("What is your monthly spend on transportation: \n");
    scanf("%d", transportation);

    char savings = income * 0.10;
    printf("%c", savings);
    
    

    return 0;

}


